cpp/src/io/parquet/predicate_pushdown.cpp

```diff
@@ -307,6 +305,9 @@ class stats_expression_converter : public ast::detail::expression_transformer {
       CUDF_EXPECTS(dynamic_cast<ast::literal const*>(&operands[1].get()) != nullptr,
                    "Second operand of binary operation with column reference must be a literal");
       v->accept(*this);
+      // Push literal into the ast::tree
+      auto const& literal =
+        _stats_expr.push(*dynamic_cast<ast::literal const*>(&operands[1].get()));
```

Comment:
Do we need to check that the dynamic_cast is not null?

Reply:
Yep, we are doing that at L305-306

Reply:
Ah. I missed that. Thanks.